tests/acceptance/CounterDialCept.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @group frontend
+ */
+
+$I = new AcceptanceTester( $scenario );
+
+$I->wantTo( 'create and check counter block - Progress Dial' );
+
+$slug = $I->generateRandomSlug() . 'counter';
+
+$I->havePageInDatabase(
+	[
+		'post_name'    => $slug,
+		'post_status'  => 'publish',
+		'post_content' => $I->generateGutenberg(
+			'wp:planet4-blocks/counter',
+			[
+				'title'       => 'Counter Block Progress Dial',
+				'description' => 'Testing counter block',
+				'style'       => 'arc',
+				'completed'   => '7000',
+				'target'      => '10000',
+				'text'        => '%completed% of %target% only %remaining% left',
+			]
+		),
+	]
+);
+
+// Navigate to the newly created page.
+$I->amOnPage( '/' . $slug );
+
+// Check title and description visible correctly.
+$I->see( 'Counter Block Progress Dial', '.counter-style-arc > div > header > h2' );
+$I->see( 'Testing counter block', '.counter-style-arc > div > .page-section-description p' );
+
+// Check progress dial element visible on page.
+$I->seeElement( 'div.content-counter svg.progress-arc' );
+$I->canSeeElement( 'div.content-counter svg.progress-arc path.foreground', [ 'stroke-dasharray' => '31.5' ] );
```

Comment:
Shouldn't this test `stroke-dashoffset` instead of `stroke-dasharray`? The `stroke-dashoffset` is the attribute that is calculated based on the progress, the `stroke-dasharray` is set to a fixed value of `31.5` in the template.